### DISCUSSION TITLE
add an option to bind certain mods to certain skins

### DISCUSSION THREAD

**Discussion by ShynxusMaximus** [ppy/osu#26476] [Ideas]
adding an option to make the game automatically change skins when using different mods (and mod combos) would be a huge quality of life improvement. for example i'd like to play the rafis skin with hddt but i won't play it on nm therefore it would be nice if it would instantly change to a nm skin. a dedicated menu in the skin settings would be nice to navigate

**Comment by peppy** [comment]
We'll be adding settings profiles in the future. Not per mod, but it should give you a starting point.
